OPEN_PROBLEMS/ENHANCED_BITSWAP_GRAPHSYNC.md

```diff
@@ -0,0 +1,152 @@
+# Enhanced Bitswap/GraphSync with more Network Smarts
+
+## Short Description
+> In one sentence or paragraph.
```

Comment:
@yiannisbot this short description has 3 paragraphs, that's what the Long description is for :)